graphqlapi/src/main/java/io/stargate/graphql/persistence/schemafirst/SchemaSourceDao.java

```diff
@@ -45,10 +47,28 @@
   @VisibleForTesting static final String APPLIED_COLUMN_NAME = "[applied]";
   private static final String DEPLOYMENT_IN_PROGRESS_COLUMN_NAME = "deployment_in_progress";
 
+  private static final Table EXPECTED_TABLE =
+      ImmutableTable.builder()
+          .keyspace("ks") // mock keyspace name, it won't be used
```

Comment:
why keyspace is not used?

Reply:
The actual keyspace will be different every time.

`CassandraSchemaHelper.compare`ignores the keyspace, and `ensureTableExists` uses its own.